LocalSettings.php

```diff
@@ -2836,6 +2836,10 @@
 			NS_FEATURED => 'Featured_Page',
 			NS_FEATURED_TALK => 'Featured_Page_talk',
 		),
+		'arphilosophywiki' => array(
```

Comment:
Need to add this to the define at the top too.

Reply:
@paladox ?

Reply:
For example "define( 'NS_TECH', 1600 );"

Reply:
Oh pfft I knew that... *slaps face with trout* (dealing with some personal issues right now so I’m not at my best)